### ISSUE TITLE
Can we have the "Zero" category back on the My Materials page?

### ISSUE THREAD

**Issue by dumdummuoi** [madcowfred/padherder-issues#320]
Or, I don't know, yet another option whether or not to show it. Personally, I use the "Zero" category for things I don't use right now but might in the future, and it's handy to know which mats I should store for that stuff.

**Comment by jarch3r** [comment]
Sounds like you should be using the Low priority.

**Comment by dumdummuoi** [comment]
No, I use low priority for things that I'm semi-actively working on. Things that I use occasionally but are not my current main focus or things that could be improved on but are usable as is. In medium priority are things that are things that need to be leveled and evolved to be used (so I put them there for the evo mat reminders) but not urgent--like alternative subs that would be situationally useful, for example. High priority is for things I want to use ASAP.

**Comment by madcowfred** [comment]
I don't think there ever was a Zero row on My Materials, though I can probably add an option to control it.
